### ISSUE TITLE
Could u please offer the requirements.txt of this code?

### ISSUE THREAD

**Issue by Shengjie-Guo** [IronSpiderMan/TongueDiagnosis#1]
Hello! IronSpiderman，i was using ur code to run some test, but my envs always cant fit your code , could u please offer the env's requirements?that would be very very nice! thank u SOOOOOO MUCH!

**Comment by IronSpiderMan** [comment]
Of course! It's juse a demo, the recommend won't be reasonable. The model's ckp file may have some problem, i can offer the original data, if you want.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "IronSpiderMan/TongueDiagnosis"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月16日(星期四) 下午2:21
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[IronSpiderMan/TongueDiagnosis] Could u please offer the requirements.txt of this code? (Issue #1)





 
Hello! IronSpiderman，i was using ur code to run some test, but my envs always cant fit your code , could u please offer the env's requirements?that would be very very nice! thank u SOOOOOO MUCH!

**Comment by Shengjie-Guo** [comment]
您好，我一直装不对对应版本的Tensorflow包和对应的Keras包，您可以提供一下您当时环境里包的版本吗？如果您能够提供训练好的参数，那就太感谢您了！

**Comment by IronSpiderMan** [comment]
我用的是tensorflow2.10和keras2.10，在实际做的时候可以不用这么高版本。邮箱上传文件太慢了，能不能提供一下一个可以接收文件的联系方式。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "IronSpiderMan/TongueDiagnosis"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月16日(星期四) 下午2:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [IronSpiderMan/TongueDiagnosis] Could u please offer the requirements.txt of this code? (Issue #1)





 
您好，我一直装不对对应版本的Tensorflow包和对应的Keras包，您可以提供一下您当时环境里包的版本吗？如果您能够提供训练好的参数，那就太感谢您了！

**Comment by Shengjie-Guo** [comment]
好的好的，sjguo@nnu.edu.cn 咱们在那里私聊吧！

**Comment by IronSpiderMan** [comment]
文件上传了比较久。这个models是模型文件，这个tongue.zip是数据文件。在我自己实验的过程中，发现用chickpoint保存的模型不一定可以用load_model方法加载。这个应该是keras的bug。所以这个模型文件在你那不一定有用




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "IronSpiderMan/TongueDiagnosis"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月16日(星期四) 下午3:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [IronSpiderMan/TongueDiagnosis] Could u please offer the requirements.txt of this code? (Issue #1)





 
***@***.*** 咱们在那里私聊吧！
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***&gt;


从QQ邮箱发来的超大附件

models.zip (257.98M, 2023年04月15日 16:13 到期)进入下载页面：https://mail.qq.com/cgi-bin/ftnExs_download?k=07626238f07284dc967fd2964532561c401606095a5401061553555a521f57060f534f085202551e5b06555a510351515900570b6338645e57060754101c1e5a48625f&t=exs_ftn_download&code=8bb8c2d3




tongue.zip (910.65M, 2023年04月15日 16:31 到期)进入下载页面：https://mail.qq.com/cgi-bin/ftnExs_download?k=5c6331336cfb2cffcd7e819d16660b171b17505756070a094e000200064b5c5c07021c0000520d150053085755030a59515a5006306c394c0c0d56465548435113630c&t=exs_ftn_download&code=cc130f98

**Comment by Shengjie-Guo** [comment]
非常非常感谢！

**Comment by joy498** [comment]
可以再把模型权重文件和模型依赖版本包的文件发一下么~

**Comment by jwolfcn** [comment]
> Of course! It's juse a demo, the recommend won't be reasonable. The model's ckp file may have some problem, i can offer the original data, if you want.
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "IronSpiderMan/TongueDiagnosis" ***@***.***&gt;; 发送时间:&nbsp;2023年3月16日(星期四) 下午2:21 ***@***.***&gt;; ***@***.***&gt;; 主题:&nbsp;[IronSpiderMan/TongueDiagnosis] Could u please offer the requirements.txt of this code? (Issue #1) Hello! IronSpiderman，i was using ur code to run some test, but my envs always cant fit your code , could u please offer the env's requirements?that would be very very nice! thank u SOOOOOO MUCH! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

the link is out of date, could u please resend me? Thanks in advance.
525079178@qq.com

**Comment by jwolfcn** [comment]
> 非常非常感谢！

可以把作者的数据文件发我一下吗，万分感谢525079178@qq.com

**Comment by IronSpiderMan** [comment]
> > 非常非常感谢！
> 
> 可以把作者的数据文件发我一下吗，万分感谢[525079178@qq.com](mailto:525079178@qq.com)

数据文件已经到邮箱了

**Comment by IronSpiderMan** [comment]
> 非常非常感谢！

如果还需要的话可以把这个项目重新整理一下

**Comment by AltonHuo** [comment]
> > 非常熱情！
> 
> 如果还需要的话可以把这个项目重新整理一下

项目很棒，就是没有环境的要求
，如果可以把环境提供出来会更好。
同求 menghuohuo@gmail.com 不胜感激

**Comment by jiangjiaqing** [comment]
你好，可否把数据发给我一份吗？非常感谢~~~ sima.lin@gmail.com （抱拳）

**Comment by justime** [comment]
谢谢， 22658318@qq.com， 非常非常感谢

> > 非常非常感谢！
> 
> 如果还需要的话可以把这个项目重新整理一下

**Comment by Alex29992896** [comment]
可以把作者的项目和数据发我一下吗，万分感谢anxinanxin2022@qq.com
